teamA/initial-settings-express/heeyeon/app.js

```diff
@@ -0,0 +1,41 @@
+const dotenv = require('dotenv');
+dotenv.config();
+
+const http = require('http');
+
+const express = require('express');
+const cors = require('cors');
+const morgan = require('morgan');
+const { DataSource } = require('typeorm');
+
+const myDataSource = new DataSource({
+    type: process.env.TYPEORM_CONNECTION,
+    host: process.env.TYPEORM_HOST,
+    port: process.env.TYPOPRM_PORT,
+    username: process.env.TYPEORM_USERNAME,
+    password: process.env.TYPEORM_PASSWORD,
+    database: process.env.TYPEORM_DATABASE
+})
+
+myDataSource.initialize()
+.then(() => {
+    console.log('Data Source has been initialized!')
+});

```

Comment:
코드의 들여쓰기를 통해 어떤 부분에 해당하는 코드인지 명시해줄 수 있습니다.

가독성을 위해 해당 부분을 아래와 같이 수정해주세요!

```suggestion
myDataSource.initialize()
    .then(() => {
        console.log('Data Source has been initialized!')
    });

```